modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/OutgoingPartitionSnapshotsCleaner.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.table.distributed.raft.snapshot;
+
+/**
+ * Stops and cleans up outgoing snapshots for a partition.
+ */
+@FunctionalInterface
+public interface OutgoingPartitionSnapshotsCleaner {
```

Comment:
This interface is not used anywhere

Reply:
This is what you get when writing code late into the night, nice catch, as always